src/main/java/io/goobox/sync/storj/UploadFileTask.java

```diff
@@ -59,11 +61,26 @@ public void run() {
         logger.info("Uploading file {}", fileName);
 
         final boolean repeat[] = { true };
+        Path tmpDir = null;
+        Path tmpPath = null;
+
+        try {
+            tmpDir = Files.createTempDirectory(Utils.getSyncDir(), ".~");
+            tmpPath = Files.createTempFile(tmpDir, "file", ".tmp");
```

Comment:
It would be better if you create the temp file in the OS temp dir instead of under the Storj sync dir. You can use this method variant: https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempFile(java.lang.String,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...)